report_thesis/src/sections/results/optimization_results.tex

```diff
@@ -0,0 +1,82 @@
+The optimization results show that the configurations with the highest total values across oxides are indicative of being the most frequently exploited, hence most likely to optimize performance. 
+\texttt{Norm3Scaler} was used in 5090 trials, and therefore appears to be the most effective scaler.
+This was somewhat expected, given the method was created for this particular type of \gls{libs} dataset, as discussed in Section~\ref{sec:norm3}.
+For dimensionality reduction, we see that \texttt{None}, indicating no \gls{pca}, was used in 9419 trials, which is \~59\% of all trials.
+This suggests that either no dimensionality reduction is optimal, or a better method should be identified.
+Neither \gls{pca} nor \gls{kernel-pca} are very effective for this dataset.
+\texttt{QuantileTransformer} was used in 5710 trials, and therefore seems to be the most optimal transformer.
+We note that \texttt{PowerTransformer} was used in 5277 trials, with and no transformer in 4956. As opposed to the other preprocessing method types, there does not seem to be a clear winner for the transformers.
+
+\input{sections/results/oxide_comparison_table.tex}
+
+We wish to examine how the top 100 trials performed for each configuration.
+We present the results in Figure~\ref{fig:top100_models}, \ref{fig:top100_scalers}, \ref{fig:top100_transformers}, and \ref{fig:top100_pca}.
+These figures and their corresponding subplots illustrate the performance of various configuration elements (models, scalers, transformers, PCA techniques) for each oxide, based on their \gls{rmsecv}.
+Any elements that do not appear in a subplot were not used in the top 100 trials for the given oxide.
+It is important to note that the variance in performance is influenced by multiple factors, not solely by the variable depicted in each plot.
+Factors such as the interaction between different preprocessing techniques, the specific hyperparameter settings, and the inherent variability in the data can all contribute to the observed performance.
+Therefore, while the plots provide valuable insights into the effectiveness of individual configuration elements, the overall performance is a result of complex interactions within the entire machine learning pipeline.
+We therefore emphasize the top performing trials and analyze a larger sample of these to draw generalizable conclusions about the optimal configurations for each oxide.
+
+From Figure~\ref{fig:top100_models}, it becomes clear that \gls{svr}, gradient boosting methods, and \gls{pls} perform the best.
+Figure~\ref{fig:top100_pca} confirms our previous hypothesis that not using any \gls{pca} or \gls{kernel-pca} results in the lowest \gls{rmsecv} values.
+We do, however, see some form of \gls{pca} or \gls{kernel-pca} appear in four of the plots, with \gls{kernel-pca} being the most frequently used of them.
```

Comment:
```suggestion
However, we do observe some form of \gls{pca} or \gls{kernel-pca} in four of the plots, with \gls{kernel-pca} being the most frequently utilized among them.
```

Reply:
Im some form does not make sense. Its either pca or kernel pca

Reply:
```suggestion
However, we do observe that either \gls{pca} or \gls{kernel-pca} appear in four of the plots, with \gls{kernel-pca} being the most frequently used among them.
```